packages/babel-preset-gatsby/README.md

```diff
@@ -0,0 +1,36 @@
+# babel-preset-gatsby
+
+Gatsby uses the phenomenal project [Babel](https://babeljs.io/) to enable support for writing modern JavaScript — while still supporting older browsers. This package contains the default Babel setup for all Gatsby projects.
+
+For more information on how to customize the Babel configuration of your Gatsby site, check out [our documentation](https://www.gatsbyjs.org/docs/babel/).
+
+## Packages
+
+- [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env)
+- [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react)
+- [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
+- [`babel-plugin-macros`](https://github.com/kentcdodds/babel-plugin-macros)
+- [`@babel/plugin-proposal-optional-chaining`](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining)
+- [`@babel/plugin-transform-runtime`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#docsNav)
+
+## Usage
+
+Install `babel-preset-gatsby` and add a `.babelrc` file with the following content to the root of your project:
+
+```bash
+npm install --dev babel-preset-gatsby
+```
+
+```json
+{
+  "presets": ["babel-preset-gatsby"]
+}
+```
+
+## Options
+
+### `targets`
+
+`{ [string]: number | string }`, defaults to `{ "browsers": ["last 4 versions", "safari >= 7", "ie >= 9"] }` in production and `{ "browsers": ["last 2 versions", "not ie <= 11", "not android 4.4.3"] }` in development when targeting the browser and `{ "node": 6 }` in production and `{ "node": "current" }` in development when targeting Node.js.
```

Comment:
Is it possible for users to configure this by add a `.browserslistrc` file? That way PostCSS can pick it up as well, which will result in a clean support table for the project.

If yes, it would be great to add this to the docs as well.

Reply:
Riiight that's documented here: https://www.gatsbyjs.org/docs/browser-support/#specify-what-browsers-your-project-supports-using-browserslist

In this case I don't think it's useful to mention this as a use case for the custom babel config like [we do right now](https://www.gatsbyjs.org/docs/babel/#how-to-use-a-custom-babelrc-file) and add the `targets` option to `babel-preset-gatsby` at all.

I'm trying to think of another use case for customizing the Babel setup but it kinda feels plugins are already doing most of the "common" stuff. Maybe we can show how to add a hypothetical `babel-transform-awesome-feature` 🙈 ?

Reply:
Actually I think it's out of scope for this PR, you've done so much already and this uncertainty isn't related to your PR. It's best to clarify this stuff in another PR.